### PR TITLE
changed main section

### PR DESCRIPTION
**Pull request by silentscript0**
changed both index.html and style.css files within main branch

### CODE REVIEW

**Review by zhpadvoiski — CHANGES_REQUESTED** [review]
take a look on my comments pls
needs some changes

File: index.html [hunk 1]
```diff
@@ -25,29 +27,92 @@
...
+
+                <div class="btn"><i class="fas fa-thumbs-up"></i> Like</div>
+                <div class="btn"><i class="fas fa-comment"></i> Comment</div>
+            </div>
+
+
+            <!-- post b -->
+            <div class="post">
+                <div class="avatar-name">
+                    <div class="avatar b"></div>
```
Comment (zhpadvoiski) [inline index.html]: you don't need class b here! think about your layout!!

**Review by zhpadvoiski — CHANGES_REQUESTED** [review]

File: style.css [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    outline-color: lightsteelblue;
+    
+}
+
+/* post a (change url Eugene)*/
+.a {
+    background-image: url(https://avatars.githubusercontent.com/u/78428105?s=460&u=b17fc119518c3430431b7fa61a12b17cbc139ea5&v=4);
+}
+
+.img-a {
```
Comment (zhpadvoiski) [inline style.css]: img should be 100%. something wrong with a layout

File: style.css [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+}
+
+.img-wrapper {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 20px;   
+}
+
+/* post b (change url Aizhana)*/
+.b {
```
Comment (zhpadvoiski) [inline style.css]: class b....?

File: style.css [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+    border: solid 1px;
+    background-image: url(https://avatars.githubusercontent.com/u/78428105?s=460&u=b17fc119518c3430431b7fa61a12b17cbc139ea5&v=4);
+}
+
+/* post c (change url Eliza) */
+.c {
```
Comment (zhpadvoiski) [inline style.css]: c??

Comment (zhpadvoiski) [reply style.css]: you should out avatar image inside a block

File: index.html [hunk 5]
```diff
@@ -25,29 +27,92 @@
...
+                    technologies. Contact us.</p>
+                <div class="img-wrapper">
+                    <img class="img-a" src="https://learn.seytech.co/assets/projects/html-css/projects/6-social-network/assets/p0.png" alt="Seytech Merch">
+                        
+                    <img class="img-a" src="https://learn.seytech.co/assets/projects/html-css/projects/6-social-network/assets/p1.jpg"
+                        alt="coding mind">
+                </div>
+
+
+                <div class="btn"><i class="fas fa-thumbs-up"></i> Like</div>
```
Comment (zhpadvoiski) [inline index.html]: buttons should be in a wrapper. If you have something similar more than one, put them in a wrapper - union

**Review by zhpadvoiski — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (silentscript0) [reply index.html]: ok, gave one class to the post photos

**Pull request merged**